libraries/AP_Airspeed/AP_Airspeed.cpp

```diff
@@ -57,7 +57,7 @@ const AP_Param::GroupInfo AP_Airspeed::var_info[] = {
     // @Description: Type of airspeed sensor
     // @Values: 0:None,1:I2C-MS4525D0,2:Analog,3:I2C-MS5525,4:I2C-MS5525 (0x76),5:I2C-MS5525 (0x77),6:I2C-SDP3X
     // @User: Standard
-    AP_GROUPINFO_FLAGS("_TYPE", 0, AP_Airspeed, param[0].type, ARSPD_DEFAULT_TYPE, AP_PARAM_FLAG_ENABLE),
+    AP_GROUPINFO_FLAGS("_TYPE", 0, AP_Airspeed, param[0].type, ARSPD_DEFAULT_TYPE, AP_PARAM_FLAG_ENABLE_MINOR),
```

Comment:
Why isn't airspeed a major item?

Reply:
its based on the likelyhood of UI refresh issues in a GCS